File: app/src/main/java/at/shockbytes/corey/storage/live/LiveScheduleUpdateListener.java
package at.shockbytes.corey.storage.live;

import at.shockbytes.corey.util.schedule.ScheduleItem;

/**
 * @author Martin Macheiner
 *         Date: 27.02.2017.
 */

public interface LiveScheduleUpdateListener {

    void onScheduleItemAdded(ScheduleItem item);

    void onScheduleItemDeleted(ScheduleItem item);

    void onScheduleItemChanged(ScheduleItem item);

}

File: app/src/main/java/at/shockbytes/corey/storage/live/LiveScheduleUpdateListener.kt
package at.shockbytes.corey.storage.live

import at.shockbytes.corey.schedule.ScheduleItem

/**
 * @author Martin Macheiner
 * Date: 27.02.2017.
 */

interface LiveScheduleUpdateListener {

    fun onScheduleItemAdded(item: ScheduleItem)

    fun onScheduleItemDeleted(item: ScheduleItem)

    fun onScheduleItemChanged(item: ScheduleItem)

}
